templates/support.html

```diff
@@ -131,7 +124,22 @@ <h2>Help fund our work</h2>
                     Foundation funds substantial work to make sure Matrix
                     addresses and will keep addressing actual issues.
                 </p>
-                <a href="#" class="call-to-action">Donate</a>
+                <p>
+                    Donorbox is our preferred method for donations because it has the lowest fees. We are keeping
+                    Patreon
+                    and LiberaPay to accommodate users preferences.
+                </p>
+                <div class="cta-row">
+                    <a class="call-to-action" href="https://donorbox.org/keep-matrix-exciting">
+                        <img src="/assets/donorbox-icon.svg" alt="" /> Donorbox
+                    </a>
+                    <a class="call-to-action" href="https://www.patreon.com/matrixdotorg">
+                        <img src="/assets/patreon-icon.svg" alt="" />Patreon
+                    </a>
+                    <a class="call-to-action" href="https://liberapay.com/matrixdotorg">
+                        <img src="/assets/liberapay-icon.svg" alt="" />LiberaPay
```

Comment:
Please dont use empty alt attributes :)

Reply:
See https://github.com/matrix-org/matrix.org/pull/2087#discussion_r1433732347

Reply:
Turns out I am wrong and this is actually good. See: https://accessibleweb.com/question-answer/when-should-i-use-a-null-or-empty-alt-tag/